src/redux-modules/settingsMiddleware.ts

```diff
@@ -76,6 +77,12 @@ export const settingsMiddleware =
         setPollTdp(activeGameId);
       }
 
+      if (action.type === suspendAction.type) {
+        serverApi.callPluginMethod(ServerAPIMethods.SET_SMT, {
```

Comment:
I'm ignoring the smt-related changes since they're in the other PR for SMT.

but I'm assuming this will eventually be removed from this PR since it's unrelated